README.md

```diff
@@ -212,13 +220,28 @@ Once the services are uninstalled, you may re-install them by running `./install
 
 *IMPORTANT*: Running the uninstall script will *NOT* remove any user data, so you can freely uninstall and re-install the services.
 
+## Migrating Package Artifacts to Minio
+
+This section is for those that have already installed an On-Premise Depot, and are now re-installing/upgrading (which will pick up the latest Builder services that have migrated over to the new artifact store).
```

Comment:
I'd also some date information here.  Something to the effect of "this section is for those that installed an on-prem depot previous to June 2018, etc".

Reply:
👍